lib/src/widgets/movie_vote_average.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '/../utils/movie_dimensions.dart';
+
+class MovieVoteAverage extends StatelessWidget {
+  final int starCount;
+  final double voteAverage;
```

Comment:
This can be a String, avoid parsing elements inside the widgets

Reply:
This cannot be String type, because I use this parameter as _double_ when I do the logic to paint the 'stars'. Line 41 of this file